yt_dlp/extractor/youtube.py

```diff
@@ -4044,6 +4051,10 @@ def _list_formats(self, video_id, microformats, video_details, player_responses,
                        else None)
         streaming_data = traverse_obj(player_responses, (..., 'streamingData'))
         *formats, subtitles = self._extract_formats_and_subtitles(streaming_data, video_id, player_url, live_status, duration)
+        if all(f.get('has_drm') for f in formats):
+            # If these are no formats that defenitely don't have DRM, all have DRM
```

Comment:
just comment typos :nerd_face:
```suggestion
            # If there are no formats that definitely don't have DRM, all have DRM
```